changelogs/CHANGELOG-v2.5.rst

```diff
@@ -1247,3 +1247,4 @@ Bugfixes
 
 - win_wait_for - fixed issue when trying to check a localport when the port is not available externally
 
+- vyos - fixed spelling of failed_conditionals json entry (https://github.com/ansible/ansible/pull/41284)
```

Comment:
This file is autogenerated from files in changelogs/fragments. No changelog entry is needed for this backport, though, since it's a very simple fix.